client/jetpack-cloud/sections/agency-dashboard/sites-overview/site-status-content.tsx

```diff
@@ -175,159 +178,174 @@ export default function SiteStatusContent( {
 		);
 	}
 
-	// We will show "Site Down" when the site is down which is handled differently.
-	if ( type === 'monitor' && ! siteDown ) {
-		return (
-			<ToggleActivateMonitoring
-				site={ rows.site.value }
-				settings={ rows.monitor.settings }
-				status={ status }
-				tooltip={ tooltip }
-				tooltipId={ tooltipId }
-				siteError={ siteError }
-				isLargeScreen={ isLargeScreen }
-			/>
-		);
-	}
+	let content;
+	let updatedTooltip = tooltip;
```

Comment:
What do you think about not introducing a new variable here?

We can do something like this:

```
const { a } = foo; 
let { b } = foo;
```

We can cache the value of `getRowMetaData` and use it on two separate destructuring assignments.